public/video-ui/src/app.js

```diff
@@ -33,7 +33,9 @@ setStore(store);
 store.dispatch({
   type:       'CONFIG_RECEIVED',
   config:     Object.assign({}, extractConfigFromPage(), {
-    embeddedMode: qs.parse(location.search).embeddedMode
+    embeddedMode: qs.parse(location.search).embeddedMode,
+    isPreviewMode: qs.parse(location.search).embeddedMode === 'preview',
```

Comment:
Do these guys get used below? Seems like we're still doing logic against the embeddedMode field